File: Mock/Data/Exception/XmlFormatErrorException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Mock.Data.Exception
{
    public class XmlFormatErrorException : TaistException
    {
        public XmlFormatErrorException(string content) : base(string.Format("Xml format error : {0}", content)) { }
    }
}

File: java/Taist.Framwork/src/Mock/Exception/XmlFormatErrorException.java
package Mock.Exception;

public class XmlFormatErrorException extends TaistException {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3046364207520609969L;

	public XmlFormatErrorException(String xmlString) {
		super("Xml format error : " + xmlString);
	}
}
